proxy/src/control_plane/client/mock.rs

```diff
@@ -119,7 +121,11 @@ impl MockControlPlane {
         Ok(AuthInfo {
             secret,
             allowed_ips,
+            // TODO
```

Comment:
I supposed there should be a bit more info or removed?

Reply:
Yes, this was a leftover!